componets/badge/badge.html

```diff
@@ -0,0 +1,180 @@
+          <h2 class="component__name">Badges on Avatar</h2>
+          <p class="component__description">
+            Badges on avatar can be used to show online offline dnd status of
+            the users, use class badge to form a badge and additionally you can
+            use online offline and dnd classes to show the status
+          </p>
+          <div class="components__example">
+            <div class="badge__container">
+              <img class="avatar s" src="../../assets/4.jpg" href="avatar1" />
+              <div class="badge bottom-right online"></div>
+            </div>
+            <div class="badge__container">
+              <img class="avatar s" src="../../assets/4.jpg" href="avatar1" />
+              <div class="badge bottom-right offline"></div>
+            </div>
+            <div class="badge__container">
+              <img class="avatar s" src="../../assets/4.jpg" href="avatar1" />
+              <div class="badge bottom-right dnd"></div>
+            </div>
+          </div>
+          <div class="component_code">
+            <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%280%2C0%2C0%2C1%29&t=material&wt=none&l=javascript&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=true&fl=1&fm=Hack&fs=14px&lh=144%25&si=false&es=2x&wm=false&code=%2520%2509%2509%253Cdiv%2520class%253D%2522badge__container%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520class%253D%2522avatar%2520s%2522%2520src%253D%2522..%252F..%252Fassets%252F4.jpg%2522%2520href%253D%2522avatar1%2522%2520%252F%253E%2520%2520%2520%2520%2520%2520%2520%2520%2520%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522badge%2520bottom-right%2520online%2522%253E%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522badge__container%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520class%253D%2522avatar%2520s%2522%2520src%253D%2522..%252F..%252Fassets%252F4.jpg%2522%2520href%253D%2522avatar1%2522%2520%252F%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522badge%2520bottom-right%2520offline%2522%253E%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522badge__container%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520class%253D%2522avatar%2520s%2522%2520src%253D%2522..%252F..%252Fassets%252F4.jpg%2522%2520href%253D%2522avatar1%2522%2520%252F%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522badge%2520bottom-right%2520dnd%2522%253E%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E"
```

Comment:
maybe shorten this URL

Reply:
I tried to shorten it but the I frame is getting disturbed like its not working how I want it to work